_posts/2023-01-26-community-manager-job-available.md

```diff
@@ -0,0 +1,147 @@
+through both mentorship and strategic partnerships with existing organizations 
+such as Metadocencia, Open Science Labs, pyLadies, geoLatinas and other 
+organizations.
+
+pyOpenSci’s core program is open peer review of scientific Python software. 
+Through that program we enforce community-defined packaging standards while 
+improving usability, documentation and package quality. Further, our goal is to 
+curate a database of vetted scientific Python tools that have ongoing 
+maintenance. 
+
+pyOpenSci is a community owned organization, fiscally sponsored by Community Initiatives.  
+
+## What we are looking for 
+We are looking for someone with experience managing large, diverse online 
+communities. We hope that you are excited about facilitating outreach activities 
+and developing community partnerships. We also want you to be committed to 
+increasing diversity in the scientific open source software community. Ideally 
+you are also familiar with the Python programming language and goals of open 
+science. 
+
+This position provides the opportunity for someone to drive the pyOpenSci 
+mission of building a diverse and supportive community forward. This is a 
+part-time position (.75-.8 FTE) with full benefits. This position will report 
+to the pyOpenSci Executive Director. 
+
+Review of applications will begin on Feb 1, 2023 and will be ongoing until the 
+position is filled. This position, and all positions in our organization, is 
+grant funded. Current funds allocated through June 2024. Assuming project 
+success during the first two years, we will seek additional funding to extend 
+the work beyond June 2024. This position is fully remote.  
+
+## What Your Key Responsibilities Will Be:
+
+### Community Management
+* Build and manage an active, diverse, and inclusive online pyOpenSci community by staying informed of the needs and motivations of current and prospective members and responding with programming that meets these needs.
+* Facilitate regular communication with pyOpenSci community members.
+* Organize, plan, and drive participation at community building events, both virtual and in-person, that provide positive and valuable interactions for members, consistent with pyOpenSci’s mission.
+* Lead diversity, equity, and inclusion (DEI) efforts to ensure that pyOpenSci members from all backgrounds are supported and empowered to participate. This will include the development of a mentorship program for underrepresented groups in open source.
+* Solicit new packages for review as well as identify and facilitate the resolution of potential hold ups in the review process.
+* Help maintain databases of community members, maintainers and packages for pyOpenSci
+
+## External Communications
+* Cultivate strategic relationships with new and existing partner organizations. Current partners include JOSS, rOpenSci, Pangeo, the Software Sustainability Institute, the Carpentries, MetaDocencia and 2i2c.
```

Comment:
maybe include links to partner orgs here?

Reply:
good call!